src/main/java/wolox/training/security/CustomAuthenticationProvider.java

```diff
@@ -0,0 +1,39 @@
+package wolox.training.security;
```

Comment:
This shouldn't be here

Reply:
Please review again, this branch wasn't update with main.